.github/mergify.yml

```diff
@@ -1,21 +1,67 @@
----
 defaults:
   actions:
     queue:
       name: default
-      method: rebase
-      update_method: rebase
-
 queue_rules:
+  - name: duplicated default from API changes needs approval from a contributor and
```

Comment:
@nixpanic I think we need to adapt the new config to the existing rule we have for API instead of having this new one, right?